EIPS/eip-2426.md

```diff
@@ -0,0 +1,141 @@
+---
+eip: 2426
+title: Encryption for Ethereum 2 walletstore and keystore
+author: Jim McDonald <Jim@mcdee.net>
+discussions-to: https://ethereum-magicians.org/t/encryption-for-ethereum-2-walletstore-and-keystore/3828
+status: Draft
+type: Standards Track
+category: ERC
+created: 2019-12-04
+requires: 2335, 2386
+---
+
+## Simple Summary
+
+An encryption and decryption method for Ethereum 2 walletstore and keystore files, along with a binary format for storing the encrypted data with the parameters required to decrypt them.
+
+## Abstract
+
+Ethereum [walletstore](https://eips.ethereum.org/EIPS/eip-2386) and [keystore](https://eips.ethereum.org/EIPS/eip-2335) files (hereafter just "stores") have various fields that some users may consider private (_e.g._ name, pubkey).  In addition, there is always the risk of stores being stolen or made publicly accessible.  To counteract potential issues this defines an encrypted format for stores.
```

Comment:
> https://eips.ethereum.org/EIPS/eip-2386

this link is a 404 for me

Reply:
Ah I see it references this: https://github.com/ethereum/EIPs/pull/2386 which is not merged yet